mvp-spec/ad-auction.md

```diff
@@ -667,9 +762,9 @@ for nom.
 
 #### The OpenRTB Bid Response
 
-In step 5, the bidder (named Receiver in PAF Transmission) send back a 
+In **step 4**, the bidder sends back a 
 OpenRTB Bid Response. Each `bid` is associated with a Transaction Response. The 
-Transaction has the same structure explained in **Step 5** and is reachable in
+Transaction has the same structure as explained in **Step 4** and is reachable in
 the `ext` field of a `bid` (full path: `seatbid[].bid.ext.paf`).
 
 Here is an example:
```

Comment:
The examples are examples of the specifications for one Seed to one TransactionId.